PowerPointLabs/PowerPointLabs/TooltipsLab/CreateTooltip.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Windows.Forms;
+
+using PowerPointLabs.ActionFramework.Common.Log;
+using PowerPointLabs.Models;
+using PowerPointLabs.TextCollection;
+using PowerPointLabs.Utils;
+using PowerPointLabs.Views;
+
+using PowerPoint = Microsoft.Office.Interop.PowerPoint;
+
+namespace PowerPointLabs.TooltipsLab
+{
+    internal static class CreateTooltip
```

Comment:
Because this is an entirely new class, I suggest adding some documentation comments to briefly describe the class if possible? `<summary> </summary> `